File: java assignment/ditel 3/ArithmeticSmallestAndLargest.java
import java.util.Scanner;
public class ArithmeticSmallestAndLargest{
	public static void main(String... args){
		Scanner collect = new Scanner(System.in);

		int sum = 0;
		int average = 0;
		int product = 1;
		int smallest = 0;
		int largest = 0;

		for(int counter = 0; counter <= 4; counter++){
			System.out.println("Enter number: ");
			int number = collect.nextInt();
	


			if(smallest == 0 || number < smallest){
	
		 		smallest = number;
			}

			if(largest == 0 || number > largest){
				largest = number;
			}
		}

	

		System.out.printf("The smallest is, %d\nThe largest is %d", smallest, largest );

	
	}
}

File: pythonassignment/Ditel 3/arithmeticsmallestandlargest.py
sum = 0
average = 0
product = 1
smallest = 0
largest = 0

for counter in range(4):
	number = int(input('Enter number: '))
	
	product *= number

	sum += number

	if smallest == 0 or number < smallest:
	
		 smallest = number

	if largest == 0 or number > largest:
		largest = number


average = sum / 2
print(f'The sum is {sum}\nThe average is, {average}\nThe product of all the numbers is {product}\nThe smallest is, {smallest}\nThe largest is, {largest}')

	